torch/distributed/_functional_collectives.py

```diff
@@ -566,6 +566,27 @@ def _all_to_all_single_meta(input, output_split_sizes, input_split_sizes, tag, r
         out_size[0] = sum(output_split_sizes)
         return input.new_empty(out_size)
 
+def _all_gather_into_tensor_native_meta(input, group_size, group_name):
+    shape = list(input.size())
+    shape[0] *= group_size
```

Comment:
I guess the group size is because of those meta functions? wondering if we can also call resolve_group inside those meta functions